### PR TITLE
add PlayerAdObserverProtocol

### PR DESCRIPTION
**Pull request by shaymark**
https://applicaster.atlassian.net/browse/IM-832

add ability to send to the PlayerDependent plugins also Ad event with a new "PlayerAdObserverProtocol"

### CODE REVIEW

File: plugins/zapp-core/apple/universal/Plugins/Player/PlayerAdObserverProtocol.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//
+//  PlayerAdObserverProtocol.swift
+//  Alamofire
```
Comment (kononenkoAnton) [inline plugins/zapp-core/apple/universal/Plugins/Player/PlayerAdObserverProtocol.swift:3]: Why is it Alamofire?

File: plugins/zapp-apple/apple/universal/Plugins/PlayerDependant/PlayerDependantPluginsManager+PlayerAdObserverProtocol.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
+//
+//  PlayerDependantPluginsManager_PlayerAdObserverProtocol.swift
+//  Alamofire
```
Comment (kononenkoAnton) [inline plugins/zapp-apple/apple/universal/Plugins/PlayerDependant/PlayerDependantPluginsManager+PlayerAdObserverProtocol.swift:3]: Why is Alamofire

Comment (shaymark) [reply plugins/zapp-apple/apple/universal/Plugins/PlayerDependant/PlayerDependantPluginsManager+PlayerAdObserverProtocol.swift]: changed to ZappPlugins

File: plugins/zapp-core/apple/universal/Plugins/Player/PlayerAdObserverProtocol.swift (hunk 1 above)
Comment (shaymark) [reply plugins/zapp-core/apple/universal/Plugins/Player/PlayerAdObserverProtocol.swift]: changed to ZappPlugins

**Pull request merged**